apps/container-registry/internal/app/main.go

```diff
@@ -220,9 +219,17 @@ var Module = fx.Module("app",
 				return c.Next()
 			}
 
+			if method == "HEAD" {
+				return c.Next()
+			}
+
+			var _user, _pass string
+
 			b_auth := basicauth.New(basicauth.Config{
 				Realm: "Forbidden",
 				Authorizer: func(u string, p string) bool {
+					_user = u
```

Comment:
should remove the _ prefix from _user, and _pass

Reply:
Fixed